### ISSUE TITLE
Community partners

### ISSUE THREAD

**Issue by sjackman** [hackseq/October_2016_private#1]
Should we contact VanBUG and ask them whether they would like to be a community partner? Though I'm not entirely sure what we would be asking of them exactly, it seems to me that they would be a useful partner for advertising the event and soliciting team leaders and participants.
http://www.hackseq.com/partners/

**Comment by adrisua** [comment]
Yes! @minisciencegirl would you like to be our liaison with VanBUG or should I contact Will?

**Comment by minisciencegirl** [comment]
Good idea @sjackman! 

I think before we approach Will and the rest of the Dev team, it's probably good to outline a list of things we would like VanBUG to do for hackseq and our expectations for VanBUG as a community partner. 
 
I deal with VanBUG's communication - so can send out emails/ announcements for hackseq when the time comes.

**Comment by adrisua** [comment]
VanBUG potential contributions:
- Help with promotion 
- Help with recruitment of participants and team leaders
- Help with recruitment of volunteers for the event (10?)
-..
Hackseq contributions:
- VanBUG logo on the website (Partners section)
- VanBUG banner at the event (if provided)
- Pitch at closing ceremony (2 min)

What do you think? @sjackman @minisciencegirl @ttimbers Something else to add?

**Comment by ababaian** [comment]
Should we distinguish 'Community Partners' like VanBug, GrasPods, SCWIST etc.. from 'Our Sponsors' like ECOSCOPE, LSI etc...

**Comment by adrisua** [comment]
Yes
